adafruit_azureiot/device_registration.py

```diff
@@ -0,0 +1,235 @@
+"""
```

Comment:
Could you include the same header used in `__init__.py` here? (reference: https://github.com/adafruit/Adafruit_CircuitPython_Wiznet5k/tree/master/adafruit_wiznet5k)

Reply:
File headers added - can you verify they are ok please.

Reply:
Look good, may want to update the copyright to be current:
`# Copyright (c) 2019 Jim Bennett`
to 
`# Copyright (c) 2020 Jim Bennett`

Reply:
D'oh! Will fix that....

Reply:
Fixed!